bears/uploadTool.py

```diff
@@ -0,0 +1,86 @@
+import argparse
+import glob
+import os
+import shutil
+from string import Template
+import subprocess
+import sys
+
+from coalib.collecting.Importers import iimport_objects
+
+
+def touch(file_name):
+    """
+    Creates an empty file. An existing file remains untouched.
+
+    :param file_name: Name of the file.
+    """
+    if not os.path.exists(file_name):
+        open(file_name, 'w').close()
+
+
+def create_file_from_template(template_file, output_file, substitution_dict):
+    """
+    Creates a file from a template file, using a substitution dict.
+
+    :param template_file:     The template file.
+    :param output_file:       The file to be written.
+    :param substitution_dict: The dict from which the substitutions are taken.
+    """
+    with open(template_file) as fl:
+        setup_py = fl.read()
+    setup_py = Template(setup_py).safe_substitute(**substitution_dict)
+
+    with open(output_file, 'w') as setup_py_handle:
+        setup_py_handle.write(setup_py)
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description='This tool uploads all the bears to PyPi in separate'
+                    'packages.')
+    parser.add_argument('-u', '--upload', help='Upload the bears on PyPi',
+                        action='store_true')
+    args = parser.parse_args()
+
+    os.makedirs('upload', exist_ok=True)
+
+    bears_full_filenames = glob.glob('**/*Bear.py')
+
+    for bear_file_name in bears_full_filenames:
+        file_name = bear_file_name
```

Comment:
sorry, you're copying the variable? Why don't you name it file name in the first place and use it consistently?